metaflow/plugins/argo/argo_workflows.py

```diff
@@ -463,6 +463,15 @@ def _process_parameters(self):
             else:
                 param_type = str(param.kwargs.get("type").__name__)
 
+            extra_attrs = {}
+            if isinstance(param.kwargs.get("type"), FilePathClass):
+                extra_attrs["is_text"] = getattr(
```

Comment:
is there a situation where  the `is_text` ends up as `True` when the File in question is not a text file?

Reply:
these are the defaults:

```
    def __init__(
        self,
        name: str,
        required: bool = False,
        is_text: bool = True,
        encoding: str = "utf-8",
        help: Optional[str] = None,
        **kwargs: Dict[str, str]
    ):
```

that situation is possible, but then, user should supply `is_text` to `False` explicitly..

Reply:
the burden of supplying the correct value of `is_text` and `encoding` is on the user..

Reply:
yeah, flow/step will have errors and will fail..

Reply:
it will be at runtime IIRC

Reply:
oh I see what you mean https://docs.metaflow.org/api/flowspec#includefile.